static/app/views/releases/components/header.tsx

```diff
@@ -63,6 +72,25 @@ function Header({router, hasV2ReleaseUIEnabled}: Props) {
           />
         </Layout.Title>
       </Layout.HeaderContent>
+      {hasV2ReleaseUIEnabled && selected === THRESHOLDS_PATH && (
+        <Layout.HeaderActions>
+          <Tooltip
+            title="Select a single project from the project filters to create a threshold for"
```

Comment:
Maybe make this more succinct? I'm not sure if it's worth including the part about project filters
```suggestion
            title="You must select a single project to create a threshold for"
```

Reply:
Since the button isn't explicitly connected to the page filters, i figured it was better to call out that out to be very clear

Reply:
For now i'll keep the verbage as is